### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by devvsakib** [devvsakib/github-error-solve#118] [Announcements]
## What's Changed
* converted to react app by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/7
* redmefile added by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/8
* json file added by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/9
* public folder added by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/10
* readme file updated by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/12
* search functionality added issue #11 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/13
* Development by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/22
* Update README.md by @HarshDeep61034 in https://github.com/devvsakib/github-error-solve/pull/23
* Development by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/24
* Development by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/25
* Development by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/30
* deleted unneccessary div by @smithop in https://github.com/devvsakib/github-error-solve/pull/27
* deleted app.js file which is not necessary by @codemeop in https://github.com/devvsakib/github-error-solve/pull/29
* Development by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/31
* added guideline how to contribute by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/32
* UI updated by @ankitk26 in https://github.com/devvsakib/github-error-solve/pull/35
* added type button by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/33
* responsive filter type menu by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/36
* Filter By Type Button Design by @anvikshik in https://github.com/devvsakib/github-error-solve/pull/42
* Adding transition delay to filter button by @hussain-shariff in https://github.com/devvsakib/github-error-solve/pull/41
* transition dur set .5s and readme updated by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/43
* Added doc to the navbar by @nawarajshah in https://github.com/devvsakib/github-error-solve/pull/44
* i have added logo issue #28 by @codemeop in https://github.com/devvsakib/github-error-solve/pull/47
* badge color and border color matched by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/45
* Changed grid layout for different breakpoints issue #38 by @imtiazpy in https://github.com/devvsakib/github-error-solve/pull/48
* added layout component,added Doc and DocItem components, wrapped App … by @imtiazpy in https://github.com/devvsakib/github-error-solve/pull/49
* Fix navbar by @imtiazpy in https://github.com/devvsakib/github-error-solve/pull/53
* Commit errors by @safil-badarudeen in https://github.com/devvsakib/github-error-solve/pull/54
* removed extra clear button by @hussain-shariff in https://github.com/devvsakib/github-error-solve/pull/60
* showMore/less by @safil-badarudeen in https://github.com/devvsakib/github-error-solve/pull/62
* new Data by @safil-badarudeen in https://github.com/devvsakib/github-error-solve/pull/64
* scrollbar removed in all browser by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/66
* doc card bg randomly change by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/67
* search type to text, clear button added issue #68 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/69
* added all type button issue #70 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/71
* pages created issue #73 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/74
* fixed title (overflow & size) by @MohamedAlDeep in https://github.com/devvsakib/github-error-solve/pull/72
* Updated the documentation by @safil-badarudeen in https://github.com/devvsakib/github-error-solve/pull/78
* doc page header added issue #76 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/80
* added discord icon in readme. by @rohinish404 in https://github.com/devvsakib/github-error-solve/pull/83
* added github icon in nav by @rohinish404 in https://github.com/devvsakib/github-error-solve/pull/86
* new error added [push] by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/87
* new error added [push] by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/88
* Add Contribution Page & Contribution Section in (README) by @MohamedAlDeep in https://github.com/devvsakib/github-error-solve/pull/84
* Project name and version change by @anvikshik in https://github.com/devvsakib/github-error-solve/pull/91
* contributors image showing, issue fixes #92 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/93
* devvsakib pp image link changed by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/96
* contributors card design improved and layout added by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/97
* Changed the name & Version by @roii31-game in https://github.com/devvsakib/github-error-solve/pull/98
* favicon added issue #99 by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/100
* branch name changed in readme by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/101
* Added new error & Removed additional data in Contibutors.json by @MohamedAlDeep in https://github.com/devvsakib/github-error-solve/pull/103
* loading added by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/106
* error card enhancement by @safil-badarudeen in https://github.com/devvsakib/github-error-solve/pull/109
* add error to error.json file by @AryanGodara in https://github.com/devvsakib/github-error-solve/pull/110
* Pull Error with Solution by @mohd-raza in https://github.com/devvsakib/github-error-solve/pull/112
* image loading skeleton adden by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/114
* feat : added twitter link inside card by @iONBain in https://github.com/devvsakib/github-error-solve/pull/115
* Updated the docs by @DaduDev in https://github.com/devvsakib/github-error-solve/pull/116
* v1.0.1 released by @devvsakib in https://github.com/devvsakib/github-error-solve/pull/117

## New Contributors
* @HarshDeep61034 made their first contribution in https://github.com/devvsakib/github-error-solve/pull/23
* @smithop made their first contribution in https://github.com/devvsakib/github-error-solve/pull/27
* @codemeop made their first contribution in https://github.com/devvsakib/github-error-solve/pull/29
* @ankitk26 made their first contribution in https://github.com/devvsakib/github-error-solve/pull/35
* @anvikshik made their first contribution in https://github.com/devvsakib/github-error-solve/pull/42
* @hussain-shariff made their first contribution in https://github.com/devvsakib/github-error-solve/pull/41
* @nawarajshah made their first contribution in https://github.com/devvsakib/github-error-solve/pull/44
* @imtiazpy made their first contribution in https://github.com/devvsakib/github-error-solve/pull/48
* @safil-badarudeen made their first contribution in https://github.com/devvsakib/github-error-solve/pull/54
* @MohamedAlDeep made their first contribution in https://github.com/devvsakib/github-error-solve/pull/72
* @rohinish404 made their first contribution in https://github.com/devvsakib/github-error-solve/pull/83
* @roii31-game made their first contribution in https://github.com/devvsakib/github-error-solve/pull/98
* @AryanGodara made their first contribution in https://github.com/devvsakib/github-error-solve/pull/110
* @mohd-raza made their first contribution in https://github.com/devvsakib/github-error-solve/pull/112
* @iONBain made their first contribution in https://github.com/devvsakib/github-error-solve/pull/115
* @DaduDev made their first contribution in https://github.com/devvsakib/github-error-solve/pull/116

**Full Changelog**: https://github.com/devvsakib/github-error-solve/commits/v1.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/devvsakib/github-error-solve/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by devvsakib** [comment]
Changes were made:

**1. UI changed**
![image](https://user-images.githubusercontent.com/88339569/221428973-ed0a90e3-4c22-40a7-a559-aaef6375cb89.png)
![image](https://user-images.githubusercontent.com/88339569/221428982-6bf39a75-c0d0-407a-9199-7f75b8c3e61f.png)

**2. Nav Link Updated**
![image](https://user-images.githubusercontent.com/88339569/221429016-c2a9f91a-f057-4d69-a434-a6db59846710.png)

**3. Doc page update*
![image](https://user-images.githubusercontent.com/88339569/221430386-ef281347-08d7-4ca2-b7f5-25f4e5239ef1.png)

**Comment by harshcodesdev** [comment]
Looks Awesome !

**Comment by mohd-raza** [comment]
@devvsakib Fantastic Brother!!!
